rootfs/etc/cont-init.d/01-adsbhubclient

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/with-contenv sh
+# shellcheck shell=sh
+
+# Check to make sure the correct command line arguments have been set
+EXITCODE=0
+if [ -z "${BEASTPORT}" ]; then
+  echo "BEASTPORT environment variable not set. Using default"
+  export BEASTPORT=30002
+fi
+if [ -z "${BEASTHOST}" ]; then
+  echo "ERROR: BEASTHOST environment variable not set"
+  EXITCODE=1
+fi
+if [ -z "${CLIENTKEY}" ]; then
+  echo "ERROR: CLIENTKEY environment variable not set"
+  echo 'Please follow the "Obtaining a client key" steps in README.md'
+  EXITCODE=1
+fi
+if [ $EXITCODE -ne 0 ]; then
+  exit 1
+fi
+
+# Set up timezone
+if [ -z "${TZ}" ]; then
+  echo "WARNING: TZ environment variable not set"
+else
+  ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime && echo "$TZ" >/etc/timezone
+fi
+
+chmod +x /usr/bin/adsbhub.sh
```

Comment:
This is a perfectly valid way of ensuring that the script is executable and I have no problems with this.

If we wanted to, we could leverage s6-overlay to do this (and keep all the permissions/modes kept in one place). See: <https://github.com/just-containers/s6-overlay#fixing-ownership--permissions>